frontend/src/hocs/withScreencast/components/VideoRecordedDialog/styled-components.ts

```diff
@@ -0,0 +1,49 @@
+import styled from "@emotion/styled"
+
+export const StyledVideo = styled.video(({ theme }) => ({
+  width: theme.sizes.full,
+  borderRadius: theme.radii.md,
+}))
+
+export const StyledDialogContainer = styled.div(({ theme }) => ({
+  display: "flex",
+  flexDirection: "column",
+  width: theme.sizes.full,
+}))
+
+export const StyledRow = styled.div(({ theme }) => ({
+  display: "flex",
+  flexDirection: "row",
+  paddingTop: theme.spacing.md,
+  paddingBottom: theme.spacing.md,
+}))
+
+export const StyledFirstColumn = styled.div(({ theme }) => ({
+  paddingRight: theme.spacing.lg,
+  textAlign: "right",
+  color: theme.colors.gray,
+  fontWeight: theme.fontWeights.bold,
+  width: "6em",
+}))
+
+export const StyledSecondColumn = styled.div(({ theme }) => ({
+  flex: 1,
+  paddingRight: theme.spacing.lg,
+}))
+
+export const StyledThirdColumn = styled.div(({ theme }) => ({
+  width: "6em",
+  [`@media (max-width: ${theme.breakpoints.sm})`]: {
+    width: 0,
+  },
+}))
+
+export const StyledVideoFormatInstructions = styled.p(({ theme }) => ({
+  marginTop: theme.spacing.sm,
+  marginBottom: theme.spacing.none,
+  fontSize: theme.fontSizes.smDefault,
```

Comment:
Currently the font size is .875 which is not any of our sizes :( But likely because this was never using our theme (changed in initial emotion PR). The font in version 0.70 is wrong but correct in this PR.

Reply:
Yea. I looked at it as a misuse of the `small` element, so I adjusted the font size. It's smaller, but I think it looks fine and is used in the intended way.